notebook/static/terminal/js/terminado.js

```diff
@@ -1,14 +1,10 @@
-define (["xterm"], function(Terminal) {
+define (["xterm", "xtermjs-fit"], function(Terminal, fit) {
     "use strict";
-    function make_terminal(element, size, ws_url) {
+    function make_terminal(element, ws_url) {
         var ws = new WebSocket(ws_url);
-        var term = new Terminal({
-          cols: size.cols,
-          rows: size.rows
-        });
+        Terminal.applyAddon(fit);
+        var term = new Terminal();
         ws.onopen = function(event) {
-            ws.send(JSON.stringify(["set_size", size.rows, size.cols,
-                                        window.innerHeight, window.innerWidth]));
```

Comment:
We'll still need to send this as well, so it knows how big the initial window is.